lib/contentful/migration.ts

```diff
@@ -17,9 +17,16 @@ export async function runMigrations(
   for (const { filePath, fileName } of pendingMigrations) {
     info(`Deploying migration ${filePath}...`)
 
-    // disable this es-lint warning because the underlying function return "any" type and we have no control over that.
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-    const result = await runMigration({ ...options, filePath, yes })
+    let result
+    try {
+      // disable this es-lint warning because the underlying function return "any" type and we have no control over that.
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+      result = await runMigration({ ...options, filePath, yes })
+    } catch (e) {
+      info(`${filePath} migration failed`)
+      error((e as Error).message)
+    }
+
```

Comment:
Another possible way would be: 

```ts
for (const { filePath, fileName } of pendingMigrations) {
    info(`Deploying migration ${filePath}...`)

    try {
      const result = await runMigration({ ...options, filePath, yes })

      if (result) {
        success(`${filePath} migration deployed`)
        migrationResult.push({ successful: true, fileName })
      } else {
        migrationResult.push({ successful: false, fileName })
      }
    } catch (e) {
      migrationResult.push({ successful: false, fileName })

      info(`${filePath} migration failed`)
      error((e as Error).message)
    }
  }
```

WDYT?

Reply:
`migrationResult.push({ successful: false, fileName })` appears twice. I don't want that. If an error occurs or a migration is skipped, result will be empty. 

I don't see how the suggestion improves existing code.

Reply:
The repetition could be solved by some boolean variable. I just don't like the eslint comments, but it is fine. 

LGTM.

Reply:
I still don't follow. Do you mind sharing more? If you say it could be solved by some boolean variable, and it fixes the es-lint warning. Can you please use the code suggestion feature here to add what you're suggesting (including the boolean variable) so this can be applied and tested?

Reply:
You are right @pmbanugo. 
The code suggestion is not fixing the issue, sorry. Next time i will execute it before. 

As I said it is just a preference and not a real bug, and it looks good to me. That is the reason I approved 😄 .

Thank you!

Reply:
@mauriciovigolo Thanks for checking